web/src/containers/ConnectedWaypoint.test.js

```diff
@@ -0,0 +1,47 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import sinon from 'sinon';
+
+import { plain as Waypoint, mapDispatchToProps } from './ConnectedWaypoint';
+
+import { scrollTo } from '../actions/navigation';
+
+describe('connected waypoint higher-order component', () => {
+  const props = {
+    id: 'id',
+    scrollTo: sinon.spy()
+  };
+
+  beforeEach(() => {
+    props.scrollTo.resetHistory();
```

Comment:
I didn't see anything that was updating `history`, so I'm curious about why this is necessary in the `beforeEach`?